### PR TITLE
Update teamviewer

### LINKED ISSUE #12234: Update: TeamViewer
### Brand Name

TeamViewer

### Official Resources for Icon and Color

- SVG: https://s7g10.scene7.com/is/content/teamviewergmbh/logo-teamviewer-2 (via website)
- Color: `#050a52`, from logo & website footer

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by faizan9233**
This change will resolve #12234 by changing the teamviewer's svg and color

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -14998,7 +14998,7 @@
         },
         {
             "title": "Spotify",
-            "hex": "1DB954",
+            "hex": "1ed760",
```
Comment (mondeja) [inline _data/simple-icons.json:15001]: You've done this change in #12252. Could you revert it here?

**Review by mondeja — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (faizan9233) [reply _data/simple-icons.json:15001]: sure i'll do

File: _data/simple-icons.json [hunk 2]
```diff
@@ -15807,7 +15807,7 @@
         },
         {
             "title": "TeamViewer",
-            "hex": "004680",
+            "hex": "050a52",
```
Comment (mondeja) [inline _data/simple-icons.json:15810]: Uppercased

```suggestion
            "hex": "050A52",
```

**Review by mondeja — CHANGES_REQUESTED** [review]

**Comment by mondeja** [comment]
Please, revert the change to Spotify.

**Pull request closed without merging**